filemojicompat-defaults/build.gradle

```diff
@@ -5,16 +5,19 @@ plugins {
 }
 
 android {
-    compileSdkVersion 32
+    namespace "de.c1710.filemojicompat_defaults"
+
+    compileSdkVersion 35
 
     defaultConfig {
-        minSdkVersion 19
-        targetSdkVersion 32
+        minSdkVersion 21
+        targetSdkVersion 35
         versionCode 5
         versionName "$VERSION_NAME"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         consumerProguardFiles "consumer-rules.pro"
+        vectorDrawables.useSupportLibrary = true
```

Comment:
This prevents the generation of pngs out of the vector drawables, which reduces artifact size from 180kB to 13kB.